python/make_zofu_driver.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/python3
```

Comment:
We may not use this. Either way, not worth reviewing this file. Copied and pasted from a different project